pkg/collector/worker/worker.go

```diff
@@ -169,9 +169,13 @@ func (w *Worker) Run() {
 
 		if sender != nil && !longRunning && config.Datadog.GetBool("integration_check_status_enabled") {
```

Comment:
@remeh Should we keep `!longRunning` condition, I'm not sure what's the impact of running `sender.Commit()` for longRunning checks 🤔